.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    branches:
+      - master
+      - stable
+name: Build
+jobs:
+  build:
+    runs-on: macos-latest
+    name: Build and deploy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gprbuild
+          path: gprbuild
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-core
+          path: gnatcoll-core
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/gnatcoll-bindings
+          path: gnatcoll-bindings
+      - uses: actions/checkout@v2
+        with:
+          repository: AdaCore/langkit
+          path: langkit
+          fetch-depth: 0
+      - run: pushd langkit; git checkout ${{ github.ref }} || true; popd
```

Comment:
Could you please:

1. Use `cd` instead of the more obscure `pushd`/`popd` ?
2. Comment this line ? I'm not sure I understand it, even though I guess it's to checkout the corresponding branch ?

TIA

Reply:
I will. This is a replacement of
```
(
    cd langkit
    python2 ../utils/travis-langkit-branch.py
)
```
It tries to download the same langkit branch as the pushed lal branch (works only for `stable` now).

Reply:
Hello Maxim,

I’m not sure I understand this part: the `travis-langkit-branch.py` script is designed to select the appropriate langkit branch for all CI situations, the `stable` branch included. So in theory it should be appropriate for GitHub actions. If you have detected a configuration where it does not behave correctly, could you give us the details?

Reply:
1. I don't have `python2` I guess
2. `travis-langkit-branch.py` uses `TRAVIS_*` variables those not provided by GH

Reply:
1. The script works fine with Python3: actually the AppVeyor build already runs it with Python3.
2. This should not be a problem: the AppVeyor build just provides the `TRAVIS_*` variables, as renamings of the `APPVEYOR_*` ones (see `utils/appveyor.sh`).

Thanks,

Reply:
This single line of code meets all my needs for now.